src/mame/heathkit/tlb.cpp

```diff
@@ -208,6 +213,42 @@ void heath_tlb_device::device_reset()
 	m_key_click_active = false;
 	m_bell_active = false;
 	m_allow_vsync_nmi = false;
+
+	ioport_value const cfg(m_config->read());
+
+	// CPU clock speed
+	switch (BIT(cfg, 0, 2))
+	{
+	case 0x01:
+		m_maincpu->set_clock(H19_3MHZ);
+		break;
+	case 0x02:
+		m_maincpu->set_clock(H19_4MHZ);
+		break;
+	case 0x03:
+		m_maincpu->set_clock(H19_6MHZ);
+		break;
+	case 0x00:
+	default:
+		// Standard Clock
+		m_maincpu->set_clock(H19_CLOCK);
+		break;
+	}
+
+	// Set screen color
+	switch(BIT(cfg,2, 2))
+	{
```

Comment:
This `switch` is missing the space between the keyword and the opening parenthesis of the control expression (unlike the one above it).

Reply:
Spacing fixed, in case you didn't notice, I have enabled the `allow edits by maintainers` in the last several PRs I've submitted.